internal/pkg/fakeroot/idtools_files.go

```diff
@@ -0,0 +1,56 @@
+//go:build !linux || !libsubid || !cgo
```

Comment:
Instead of deleting idtools_unsupported and adding idtools_files, rename the former to the latter before deleting getMappingEntries and adding getIDRange.

Reply:
hmm, I acutally renamed the file and did not delete it. somehow the github shows the renaming as a deletion and addition? I renamed the file directly in vscode. 
https://github.com/orgs/community/discussions/43177

Reply:
I had the same problem with using the git cli.  Oh well.